### PR TITLE
Mail reporting

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by ghost** [comment]
Basic mail reporting for receiving a weeky/monhtly update on all transactions

**Review by Broxibear97 — APPROVED** [review]
Looks good to me

File: composer.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     "require": {
         "php": ">=7.0.0",
         "fideloper/proxy": "~3.3",
+        "guzzlehttp/guzzle": "^6.3",
```
Comment (Broxibear97) [inline composer.json:10]: What is Guzzle being used for?

Comment (ghost) [reply composer.json:10]: I believe it's for interacting with the SMTP server

Comment (ghost) [reply composer.json:10]: I think it's added automatically via the php artisan make:mailable something

**Pull request merged**